src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -113,7 +113,7 @@ public void ConfigureServices(IServiceCollection services)
 #else
             services.AddControllersWithViews();
 #endif
-#if (OrganizationalAuth || IndividualAuth)
+#if (OrganizationalAuth || IndividualB2CAuth)
```

Comment:
@jmprieur Here too, I saw our test for IndividualLocalAuth fail here. Looking through the logic, I think this only applies to IndividualB2CAuth.